cmd/fissh/main.go

```diff
@@ -65,6 +67,16 @@ func main() {
 	}
 }
 
+func extractTimezoneFromEnv(env []string) (string, error) {
+	idx := slices.IndexFunc(env, func(s string) bool { return strings.HasPrefix(s, "TIMEZONE=") })
```

Comment:
I think the standard variable name here is `TZ`? Might be nice so we can pick up on the user's env vars if they're there

Reply:
Sounds good!